src/OpenSage.Game/Content/MapLoader.cs

```diff
@@ -400,7 +403,8 @@ private static GameObject CreateGameObject(MapObject mapObject, Team[] teams, Co
                         
                         break;
 
-                    default:
+                    case RoadType.Start:
```

Comment:
Is this just a syntactical clarification, or does it change behaviour in some way?

Reply:
It's a syntactical clarification. Since there are still 2 unknown chunks i wanted to use the chunk names explicitly